README.md

```diff
@@ -32,3 +32,13 @@ You can edit the exclude file before execution:
 - All lines starting with a `#` are ignored by rsync, i.e. those directories will be backed up.
 - The syntax doesn't support comments at the end of a line yet.
 - At the start there is a section with directories that are probably not worth backing up. Uncomment those lines to exclude them as well.
+
+## Making incremental backups:
+When running locally or with the `--whole-file` option, rsync doesn't modify files but replaces them entirely.
+This allows us to snapshot the state of the backup directory at a certain point in time:
+
+    BACKUPDIR=/media/workspace/home/$USER/
+    SNAPSHOT_DIR="$BACKUPDIR.$(date --iso-8601=seconds -u)"
+    cp -al $BACKUPDIR $SNAPSHOT_DIR
+
+Next time you run your backup, the snapshot directory will be intact despite the changes rsync makes to the files in the backup directory.
```

Comment:
This sentense is still unclear. how often do you call the cp command and when? When do you call the main rsync then? and on which folder? why is the snapshotdir inside the backupdir? 

please add the rsync command in this section too as example

Reply:
Run this after you finish your backup and you'll have your snapshot.